R/zzz.R

```diff
@@ -207,6 +167,28 @@ default_undesirable_functions <- all_undesirable_functions[names(all_undesirable
   NULL
 )]
 
+rd_auto_link <- function(x) {
+  x <- unlist(x)
+  x <- gsub("withr::([a-zA-Z0-9._]+)\\(\\)", "\\\\code{\\\\link[withr:\\1]{withr::\\1()}}", x)
```

Comment:
Maybe capture the package in a group so other packages, if used in the future, are supported?

Reply:
Hmm I intentionally didn't do that, for a reason I now forget. Maybe I've just improved the other regex enough to avoid overlap. Fixed.